examples/transformerconfigs/README.md

```diff
@@ -606,3 +606,63 @@ metadata:
     foo.k8s.io/bar: baz
 ```
 Kustomize supports escaping special characters in path, e.g `metadata/annotations/foo.k8s.io\/bar`
+
+## Namespace
+
+The namespace transformer is used to update namespace references in your manifests when you set the `namespace` declaration in your `kustomization.yaml`.
+
+### Example
+
+kustomization.yaml
+
+```
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: example
```

Comment:
I think if we add a new line before `namespace: example`, this example is easy to understand.